portafly/src/components/data-list/DataListContext.tsx

```diff
@@ -28,7 +28,9 @@ const defaultState: State = {
     rows: [],
     columns: []
   },
-  filters: {},
+  filters: {
+    categories: []
```

Comment:
Hmmm, I'm not convinced about this. Previously, filters were `Record<string, string[]>` were the first string corresponds to a `Category`. If you want to include the categories in the state, I'd rename the `filters` to something else to include both `filters` and `categories`. Also, those could be in different slices of the state if it makes sense.

Reply:
Well using categories is a middle step to find the column index so I think it could be a more elegant solution to add an "id" to the columns, where such id is the name of the category (see last commit).